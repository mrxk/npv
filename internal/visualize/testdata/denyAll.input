---
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: denyAll
  namespace: default
spec:
  podSelector: {}
  policyTypes:
    - Ingress
    - Egress
