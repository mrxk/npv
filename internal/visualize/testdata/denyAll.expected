@startuml
left to right direction
frame Pods {
component "Name: denyAll\lNamespace: default\lAll\l" as default_ALL_ {
    port "0-65535" as default_ALL__ALL_port
    portout " " as default_ALL_portout
}
}
frame Ingress {
component "ALL" as _ALL_PEER_INGRESS__i {
    portout " " as _ALL_PEER_INGRESS_ingressportout
}
}
_ALL_PEER_INGRESS_ingressportout --down[#red]--> default_ALL__ALL_port
frame Egress {
component "ALL" as default_ALL__ALL__e {
    port "0-65535" as default_ALL__ALL_egressport
}
}
default_ALL_portout --down[#red]--> default_ALL__ALL_egressport
@enduml
